content/extend/gorma.md

```diff
@@ -6,3 +6,233 @@ title = "the gorma Plugin"
+
+```go
+	Store("postgres", gorma.Postgres, ...)
+```
+Today these identifiers aren't used in code generation, but may be used in the future to change the generated code in database specific ways.  The name isn't special, you can use any name you like.
+
+The example Store above has a `Description` and several `Model` definitions:
+
+```go
+	Store("postgres", gorma.Postgres, func() {
+		Description("This is the Postgres relational store")
+		Model("User", func() { ...
+```
+
+Your Store may contain an arbitrary number of models, and the models you specify aren't limited to the ones that are specified in your goa DSL.
+
+### Model
+A Model represents a single `struct` or domain object that maps to a database table.  The Model DSL is the most flexible DSL in `gorma` and it's where you'll do most of your specification.  Models generated by `gorma` use [gorm](https://github.com/jinzhu/gorm) to access your database.  This means that `gorma` supports any database driver that gorm supports.  All testing in `gorma` is currently done in PostgreSQL.  `gorma` uses no database specific functionality, so it should work equally well across supported databases.
+
+Let's break down the Model definition and see what's happening:
+
+```go
+		Model("User", func() {
+			BuildsFrom(func() {
+				Payload("user", "create")
+				Payload("user", "update")
+			})
+			RendersTo(User)
+			Description("User Model Description")
+			HasMany("Reviews", "Review")
+			HasMany("Proposals", "Proposal")
+			Field("id", gorma.Integer, func() {
+				PrimaryKey()
+				Description("This is the User Model PK field")
+			})
+			Field("created_at", gorma.Timestamp, func() {})
+			Field("updated_at", gorma.Timestamp, func() {})
+			Field("deleted_at", gorma.NullableTimestamp, func() {})
+		})
+
+```
+The first stanza in the configuration is the `BuildsFrom` DSL.  This `BuildsFrom` tells `gorma` that this model will accept data that is input from the payloads that are sent to the "create" and "update" actions of the User resource.  This is important because `gorma` will automatically generate conversion functions that convert goa's strongly typed Payload objects into a User model suitable for the Create and Update methods in the data access code.
+
+The next line specifies that the User model `RendersTo` the "User" media type in your goa design.  Again, `gorma` will automatically generate conversion functions for you that convert the database model into strongly typed goa Views. Because of the power of the goa DSL `gorma` is able to generate code to create the full view object graph.  If your User media type specifies that it will return goa Links, `gorma` will generate the code to create and embed those links in your view.  Similarly, if your view specifies that it returns nested models, `gorma` will retrieve and populate those models too.  Here's an example of a goa Media Type that specifies embedded objects in it's definition:
```

Comment:
Comma: s/Because of the power of the goa DSL/Because of the power of the goa DSL,/

s/it's/its/ in the last sentence.